tests/baseline/test_baseline.py

```diff
@@ -77,6 +81,24 @@ def koji_test_backend(fake_koji, koji_dir):
     Source.reset()
 
 
+@pytest.fixture(autouse=True)
+def fake_kerberos_auth(mocker):
+    mocker.patch("gssapi.Name")
+    mocker.patch("gssapi.Credentials.acquire")
+    mocker.patch("requests_gssapi.HTTPSPNEGOAuth", return_value=None)
+
+
+@pytest.fixture(autouse=True)
+def errata_requests_mock(requests_mock):
+    with open(ERRATUM_DATA_FILE, "r") as f:
```

Comment:
I added a fixture that automatically loads all .json data to requests_mock. It's quite rudimentary as it only supports one query type. It can be extended and modified when pushsource is switched to use Errata HTTP API.

I didn't want to mix the HTTP data with data meant for the XMLRPC client. Especially since the latter will be removed soon.

I hope it's good enough as a temporary solution.